packages/transaction-controller/src/helpers/GasFeePoller.ts

```diff
@@ -112,9 +124,10 @@ export class GasFeePoller {
     });
 
     await Promise.all(
-      unapprovedTransactions.map((tx) =>
+      unapprovedTransactions.flatMap((tx) => [
         this.#updateTransactionSuggestedFees(tx),
```

Comment:
Not your fault, but can we rename this for clarity to `#updateTransactionGasFeeEstimates` as we renamed the property after this method was named.

Reply:
Done in https://github.com/MetaMask/core/pull/3944/commits/7559973ab4a977ba70a4e00661ec8249cc9387ab